assigner/config.py

```diff
@@ -22,6 +24,12 @@ class Config(UserDict):
 
         "type": "object",
         "properties": {
+            # Backend type (gitlab / mock)
+            "backend": {
+                "type": "string",
+                "pattern": "^(mock|gitlab)?$"
+            },
+
```

Comment:
Oh, we need to add some code to automatically migrate existing configs to new ones. I get the sense it's about time that we started versioning config files. That'd probably be good to do as a separate PR first?

Reply:
I'll take this on unless you really want to -- I have a few things I'd like to improve about the current state of config handling in general.

Reply:
Please, go ahead. I'm moving the next few days so I'll be not here.